sponsors.js

```diff
@@ -96,12 +96,16 @@ function loadOneSilverSponsor(num, elem, scale) {
         case 5:
             showOneSponsor(elem, "http://generalyo.com/", "sponsors/generalyo.png", "GeneralYo", 250*scale);
             break;
+        case 6:
+            showOneSponsor(elem, "https://www.lathedbackdesign.com/", "sponsors/lathedbackdesign.png", "Lathed Back Design", 210*scale);
```

Comment:
Square aspect ratio, so it should be same size as Raincity

Reply:
Scale fixed, gotta merge the branch now.